src/main/resources/database/initDB.sql

```diff
@@ -1,9 +1,8 @@
-CREATE DATABASE litequizappdb;
-CREATE USER postgres WITH password 'admin';
-GRANT ALL privileges ON DATABASE litequizappdb TO postgres;
-
 CREATE TABLE IF NOT EXISTS categories
 (
     id     BIGSERIAL PRIMARY KEY NOT NULL,
     title  VARCHAR(100) NOT NULL
-);
\ No newline at end of file
+);
+
+CREATE SEQUENCE hibernate_sequence START 1;
+SELECT setval('hibernate_sequence', (SELECT max(id) FROM public.categories));
```

Comment:
If it fixes the problem with liquibase i'd suggest to remove commented line in compose file.

Reply:
done